src/native-common/Button.tsx

```diff
@@ -128,11 +128,18 @@ export class Button extends React.Component<Types.ButtonProps, {}> {
             _defaultAccessibilityTrait);
 
         const opacityStyle = !this.props.disableTouchOpacityAnimation && this._opacityAnimatedStyle;
+        let disabledStyle = this.props.disabled && _styles.disabled;
+
+        if (this.props.disabled && this.props.disabledOpacity) {
```

Comment:
I think the check should be this.props.disabledOpacity !== undefined. As it's written, the conditional will evaluate to false if someone explicitly sets the disabledOpacity to 0, which seems like a valid (although not recommended) action.

Reply:
Yes will do - thanks